### DISCUSSION TITLE
Export data?

### DISCUSSION THREAD

**Discussion by thelumlaa** [Azure/ipam#195] [Q&A]
We have a central NeDi system for tracking WAN endpoints. Now that we have Azure IPAM, we'd like to export data from it to feed it to NeDi. Is there a built-in function to export data from Azure IPAM?

**Comment by DCMattyG** [comment]
Hi @thelumlaa, can you help me understand a bit more as to the data that you like to export and the format you'd want to see it in?

Are we talking about a CSV export, or dumping out a large JSON file?

Let me know what you're looking for and we'll see what we can do!

**Comment by thelumlaa** [comment]
Hi @DCMattyG ! I think a CSV would be best, but the NeDi support guys said that pretty much any format would do the trick.
I dug around and found some info in cosmosDB, however it seems to not show the details all the way down to endpoints. Would it be possible to get a hold of the detailed information?
